example/cabal.project

```diff
@@ -0,0 +1,64 @@
+packages:
+    ./
+
+source-repository-package
```

Comment:
We could really do with https://github.com/haskell/cabal/issues/5472...

Reply:
I doubt this type of thing will be in cabal 2.x ever, I imagine we will have to wait for cabal-install 3